testssl.sh

```diff
@@ -1334,7 +1334,7 @@ html_header() {
           html_out "<?xml version=\"1.0\" encoding=\"UTF-8\" ?>\n"
           html_out "<!DOCTYPE html PUBLIC \"-//W3C//DTD XHTML 1.0 Strict//EN\" \"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd\">\n"
           html_out "<!-- This file was created with testssl.sh. https://testssl.sh -->\n"
-          html_out "<html xmlns=\"http://www.w3.org/1999/xhtml\">\n"
+          html_out "<html xmlns=\"http://www.w3.org/1999/xhtml/\">\n"
```

Comment:
Hmm... for some reason the w3c even lists without trailing slash: https://www.w3.org/TR/1999/WD-xhtml1-19991124/ .  Not probably that it would matter much but how come you stumbled over this (reference)?

Reply:
I do not remember messing with this, just a mistake probably. Sorry.